test/framework/src/main/java/org/elasticsearch/test/BreakerTestUtil.java

```diff
@@ -53,7 +53,7 @@ public static <E extends Exception> ByteSizeValue findBreakerLimit(ByteSizeValue
         try {
             c.accept(onePastLimit);
         } catch (CircuitBreakingException e) {
-            throw new IllegalArgumentException("expected runnable to break under a limit of " + onePastLimit + " bytes");
+            throw new IllegalArgumentException("expected runnable to not break under a limit of " + onePastLimit + " bytes");
```

Comment:
Super nit - for consistency with the above messages

```suggestion
            throw new IllegalArgumentException("expected runnable to *not* break under a limit of " + onePastLimit + " bytes");
```